charts/rancher-webhook/templates/validate-psp-install.yaml

```diff
@@ -0,0 +1,5 @@
+#{{- if .Values.global.cattle.psp.enabled }}
+#{{- if not (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
+#{{- fail "The target cluster does not have the PodSecurityPolicy API resource. Please disable PSPs in this chart before proceeding." "" -}}
```

Comment:
Sorry, wasn't super clear here. `fail` will only take one arg, where as required took two. This is causing your CI to fail. I would recommend taking the content of this file directly from the issue.